incubator/nodejs-loopback/templates/scaffold/.yo-rc.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
this is file needed?

Reply:
Updated. It tracks the `@loopback/cli` version.